src/app/components/secret-key/two-column.layout.tsx

```diff
@@ -26,7 +26,9 @@ export function TwoColumnLayout({
         gap="space.07"
         mx={['auto', 'space.03', 'space.03', 'space.03']}
       >
-        <styled.div maxWidth="440px">{leftColumn}</styled.div>
+        <styled.div minWidth="440px" maxWidth="440px" textAlign="left">
```

Comment:
Would setting the min and max width here as the same be just a fixed width of `440px`? 🤔

Reply:
Thanks @fbwoolf 🤦 the worst part is I saw this but thought I was avoiding a set width. I'll update it